lib/cocoapods/generator/xcconfig/xcconfig_helper.rb

```diff
@@ -312,6 +312,32 @@ def self.settings_for_dependent_targets(target, dependent_targets, test_xcconfig
           build_settings
         end
 
+        # Updates xcconfig with the HEADER_SEARCH_PATHS from the search_paths.
+        #
+        # @param  [Target] search_paths_target
+        #         The target.
+        #
+        # @param  [Xcodeproj::Config] xcconfig
+        #         The xcconfig to edit.
+        #
+        def self.header_search_paths_from_search_paths(search_paths_target, xcconfig)
```

Comment:
this method probably should have a verb, since it updates/adds the search paths rather than returning them